scripting/include/get5.inc

```diff
@@ -81,6 +81,26 @@ forward void Get5_OnMapVetoed(MatchTeam team, const char[] map);
 // and not by one of the teams specifically.
 forward void Get5_OnMapPicked(MatchTeam team, const char[] map);
 
+// Called when a player is elected the MVP of the round.
+forward void Get5_OnPlayerBecameMVP(int mapNumber, int roundNumber, int client, int clientSide, int reason);
+
+// Called when a player is killed in a live match. Used by MySQL stats. roundTime is milliseconds.
+forward void Get5_OnPlayerDied(char[] weapon, bool headshot, int mapNumber, int roundNumber, int roundTime,
+    int attacker, int victim, int assister, bool flashAssist, int penetratedObjects, bool thruSmoke, bool noScope,
+    bool attackerBlind, int attackerSide, int assisterSide, int victimSide);
+
+// Called when a round starts. Used by MySQL stats.
```

Comment:
No need for comments like "Used by MySQL stats", imo. None of the public API being exposed here is meant to be specific to any particular use.